### DISCUSSION TITLE
家人们好 新人报到 我是简杜的狗

### DISCUSSION THREAD

**Discussion by Yulooooo** [share121/inter-knot#2600] [其他]
简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜简杜

**Comment by hua-yuanshuai** [comment]
我是雅小姐的狗
星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅星见雅

**Reply by Yulooooo** [reply]
啊你..

**Comment by hua-yuanshuai** [comment]
怎么了
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/eb427fe8d802a27e82e694e7ee400bc3_7459590478493659790.gif" alt="符华-变身" width="75" >
